src/modules/Thread/context/ThreadProvider.tsx

```diff
@@ -222,7 +218,7 @@ export const ThreadProvider = (props: ThreadProviderProps) => {
 
   // memo
   const nicknamesMap: Map<string, string> = useMemo(() => (
-    (replyType && currentChannel)
+    (config.groupChannel.replyType !== 'none' && currentChannel)
```

Comment:
can't it be undefined?

Reply:
Yes, @HoonBaek Could you confirm what you wanted to check in this logic during thread development?

Reply:
Yes, it's default value has been 'NONE'